oeps/oep-0010.rst

```diff
@@ -0,0 +1,202 @@
+
+- An engineering team either in edX or outside of edX, must agree to be
+  minimally available for answering questions in the public channels.
+
+- An engineering team either in edX or outside of edX, must agree to provide
+  fixes if severe problems are identified.
+
+Many of these criteria are open to interpretation, or varying degrees of
+effort. For example, what does "documented thoroughly" mean? We can't quantify
+that. It and other loose criteria are included here because they are important
+parts of providing a finished quality product, and we don't want to overlook
+them.
+
+
+Release creation
+----------------
+
+A new release line is created every four to six months.  Release lines are
+named with words in alphabetical order: Dogwood, Eucalyptus, Ficus, Gingko,
+Hawthorn.  On a release line, there will be a handful of releases:
+Eucalyptus.1, Eucalyptus.2, and so on.
+
+After the .1 release, new releases in a line are made only for severe problems
+such as security problems, data loss, or feature breakage. 
+
+A new release line is created by making a "release master" branch in each
+involved repo.  These are named "open-release/RELEASENAME.master".  This branch
+will be where changes are accumulated to create each release in the line.
+Releases will be tagged "open-release/RELEASENAME.1",
+"open-release/RELEASENAME.2", and so on.
+
+
+Involving repos in the Open edX build process
+---------------------------------------------
+
+:doc:`OEP-2 <oep-0002>` defines a file format for repository metadata.  The 
+``openedx-release`` key is an optional dictionary governing the participation
+of the repo in the Open edX release process.
+
+``openedx-release``: dictionary (optional)
+
+    Possible keys:
+
+    ``ref``
+        The name of the release-from branch in this repo. This is the branch
+        that will be tagged when an Open edX release is made.
+
+    ``requirements``
+        The path to a requirements.txt-style file in this repo. This file will
+        be consulted to find the commit to tag in dependent repos indicated by
+        ``parent-repo`` keys.
+
+    ``parent-repo``
+        The name of another repo, one that installs this repo via a
+        requirements file.  The parent repo's requirements file will indicate
+        the commit to tag in this repo.
+
+        .. note::
+            We believe we no longer need to tag dependent repos, so this may go
+            away.
+
+
+
+Installing repos
+----------------
+
+Our installations are based on Anisble playbooks.  Up until the Eucalyptus
+release, all three supported installation methods were single-machine: all of
+the Open edX software was installed and ran on a single machine, either a
+Virtualbox image, or a native machine.
+
+That model does not scale up as the number of services and applications grows.
+Newer services are supporting Docker for installation.  Eventually, we would
+like to be in an all-Docker world where an installation is just a constellation
```

Comment:
Even if it isn't a supported method, others might have the same question reading this. If we state the long-term plan, it seem a sane assumption that this would then also apply to production setups? If not, it would be clearer (and more maintainable) to make the distinction explicitly, and specify what the difference would be.